jazz-ui/src/app/pages/environment-deployment/env-deployments-section.component.ts

```diff
@@ -619,13 +618,11 @@ toast_pop(error,oops,errorMessage)
 
 rebuild(){
   this.rowclick = false;
-  this.http.post('/jazz/deployments/'+this.rebuild_id+'/re-build').subscribe(
+  var rebuild_url = '/jazz/deployments/';
```

Comment:
use it directly if you are not using the variable

Reply:
made the changes.